extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegesBuilder.scala

```diff
@@ -102,6 +103,15 @@ object PrivilegesBuilder {
         val cols = conditionList ++ aggCols
         buildQuery(a.child, privilegeObjects, projectionList, cols, spark)
 
+      case pvm: PermanentViewMarker =>
+        getScanSpec(pvm).tables(pvm, spark).foreach { table =>
+          if (pvm.child.output.isEmpty) {
+            privilegeObjects += PrivilegeObject(table, pvm.catalogTable.schema.map(_.name))
```

Comment:
Sounds like it's asking for the privilege on all the columns of the table.
But what if a user with privilege on single or part of columns, and this would prevent him from counting the table.
WDYT ? @yaooqinn